src/HostnetExperimental/ruleset.xml

```diff
@@ -2,22 +2,4 @@
 <ruleset name="HostnetExperimental">
```

Comment:
I don't think this rule set does anything anymore? Seems like it could be removed completely.

Reply:
As discussed, the file has to be kept for future reference.